packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -925,33 +932,27 @@ Tooltip.propTypes /* remove-proptypes */ = {
   ]),
   /**
    * The component used for the popper.
-   * @default Popper
+   * @deprecated use the `slots.popper` prop instead. This prop will be removed in v7. [How to migrate](https://mui.com/material-ui/migration/migrating-from-deprecated-apis/).
```

Comment:
We reworded `"How to migrate"` to `"Migrating from deprecated APIs"` as it reads better:

```suggestion
   * @deprecated use the `slots.popper` prop instead. This prop will be removed in v7. See [Migrating from deprecated APIs](https://mui.com/material-ui/migration/migrating-from-deprecated-apis/) for more details.
```

Same with the others